addons/product/views/product_attribute_views.xml

```diff
@@ -18,8 +18,15 @@
         <field name="model">product.attribute</field>
         <field name="arch" type="xml">
             <form string="Product Attribute">
+            <field name="active" invisible="1"/>
```

Comment:
Most fields needed for invisible/required/readonly/... fields are now added automatically by the orm.  Did you test without this line?